desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Customers and Employees union.analysis.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<job xmlns="http://eobjects.org/analyzerbeans/job/1.0">
+	<job-metadata>
```

Comment:
I removed the metadata about the author, timestamp etc., but left the coordinates of the components on the canvas.